src/SFML/Window/SensorManager.hpp

```diff
@@ -136,7 +145,8 @@ class SensorManager
     ////////////////////////////////////////////////////////////
     // Member data
     ////////////////////////////////////////////////////////////
-    Item m_sensors[Sensor::Count]; //!< Sensors information and state
+    Item m_sensors[Sensor::Count]; ///< Sensors information and state
+    static SensorManager* singleInstance; ///< The single instance of the manager
```

Comment:
Why not use `unique_ptr` to avoid the `new` and `delete` calls? Also, why not put this static variable in SensorManager.cpp since that's the only file which needs it?

Reply:
I guess the original reason is that back when I wrote this, SFML didn't support C++11. So probably we can switch now.

To me, it just felt like it belongs to the class. Would it work outside of the class in the same way?

Reply:
> I guess the original reason is that back when I wrote this, SFML didn't support C++11. So probably we can switch now.

I think you're right. In 2019 this wasn't an option for you since that was before SFML 3 was being developed.

> To me, it just felt like it belongs to the class. Would it work outside of the class in the same way?

As far as I'm aware this would work just fine as a global in the .cpp file. It's worth testing though in case I'm missing something.

Reply:
Alright, I'll do both changes once I know if the PR is good to go otherwise the changes you mentioned.